### ISSUE TITLE
<Bug>  ZFS Storage Pool not created

### ISSUE THREAD

**Issue by Meywether** [AdaptiveScale/lxdui#170]
Dear programmers,
after experimenting with your great programm, I wanted to create a ZFS volume.
The GUI did not give any response after filling out the form Driver and Storage Pool name.

This was the error message within the terminal:
![image](https://user-images.githubusercontent.com/39849054/40959454-04425470-689d-11e8-8261-20ea291be728.png)

According to the official LXD Documentation I was able to locate the problem in more detail:

[https://lxd.readthedocs.io/en/latest/storage/](https://lxd.readthedocs.io/en/latest/storage/)

The main problem was ZFS does not need a directory path to be mounted. It needs a device path,

I created a ZFS Pool by using : `lxc storage create pool1_zfs source=/dev/sdb`

But afterwards it was not listed within the GUI Tool.  By using `lxd storage list` it was visible to the system: 
![image](https://user-images.githubusercontent.com/39849054/40959768-fc9d3b08-689d-11e8-8249-dc4aa08fb736.png)

So it seems that an input field to the device is needed for zfs pools.

Hopefully my description was helpful!

Best regards

Meywether

******************************************************* UPDATE 06.06.2018

Hi programmers,
One additional information:
If I want to add a zfs pool manually it does not show up in the UI Tool.

If I add a normal pool (e.g. dir) then it shows up in the UI Tool as well.

Additionally I strongly suggest an input field to create storage pools by providing the source property. So the creation of pools outside of /var/lxc is possible e.g. at /containers. It is possible to crete the storage pools manually via cli where I want but it really would be nice to do it via the UI!

Thanks in advance

Meywether

**Comment by vhajdari** [comment]
Hi @Meywether, thanks for your feedback.  We tested a number of scenarios locally with the 3.1 LXD Snap package and did not run into the same problem. 

To help us test this scenario further the following would be helpful:
Are you using the LXD snap or the apt package?  
What version of LXD are you running?
Which version of Ubuntu are you using?

**Comment by Meywether** [comment]
Hi vhajdari,
I am using Ubuntu 18.04 with latest updates.
I ran into problems by using LXD from your snap image at first.
Now I reinstalled an clean ubuntu 1804 and try your snap image, purging the original apt lxd first.
Both ways are running as expected by using the cli. 
Even the default created ZFS Volume shows up in the UI now. But creating a new one fails as expected! 

![image](https://user-images.githubusercontent.com/39849054/41145905-d880f2bc-6b01-11e8-94a3-7e9de72530f4.png)

To summarize:
Ubuntu: 18.04 with updates @ 08.06.2018
LXD Version: 3.0.1
LXDUI: Branch develop cloned @08.06.2018

Used LXD: Apt package and SNAP Package both showing up the same error!

Best regards
Meywether

**Comment by ailegion** [comment]
issue will be closed as it could not be replicated
